src/components/pastAnalysis/seriesTable.tsx

```diff
@@ -47,7 +47,7 @@ const SeriesTable: React.FC<SeriesTableProps> = ({ studyInstance, isProcessing }
 
   const rows = analysisList.map((analysis: ISeries, index: number) => {
     let analysisCells: any = [
-      { title: (analysis.imageUrl ? <div><b><img src={analysis.imageUrl} className="thumbnail" /></b></div> : <div><PreviewNotAvailable /></div>) },
+      { title: (analysis.imageUrl ? <div><img src={analysis.imageUrl} className="thumbnail" alt="Analysis Scan"/></div> : <div><PreviewNotAvailable /></div>) },
```

Comment:
Can you change `alt` to "Analysis Scan Thumbnail", I believe it aligns better since describes a thumbnail.

Reply:
Great suggestion, implemented!